atoma-state/Cargo.toml

```diff
@@ -5,11 +5,13 @@ edition.workspace = true
 license.workspace = true
 
 [dependencies]
+atoma-confidential = { workspace = true, features = ["sev-snp", "tdx"] }
```

Comment:
had to include them both here - and this is what throws compilation issues when doing `cargo c` etc. on a macb, because the sev-snp crate is feature-gated for Linux.